src/components/ConnectWallet/Wallets/Sparrow.vue

```diff
@@ -21,10 +24,6 @@
       <step>
         After verifying, click <span class="font-weight-bold">"Close"</span>.
       </step>
-      <step>
-        At any moment, you can turn on or off the connection using the switch 
-        in the bottom right of the window.
-      </step>
```

Comment:
Why remove this? Maybe rephrase it? 

Thinking about it, maybe Sparrow already says on first run?

Reply:
Because it's a wallet connector and not a wallet disconnector 😝

Haha, but seriously I don't think we need to add it because:

1. The purpose of wallet connector is to help user connect their wallet to Umbrel, so we don't need to add steps that explain how to disconnect from Umbrel.
2. Consistency: we don't have any such step that explains how to disconnect for any other wallet, so we don't need to add it here too.

Reply:
I'm not adding this here to tell the user how to disconnect the node, but it might help users to troubleshoot their Sparrow not connecting to Umbrel. It is possible to click on this switch by mistake and disconnect the external node.